### DISCUSSION TITLE
PWAScript is not runing when site is pwa enabled

### DISCUSSION THREAD

**Discussion by qunshen** [oqtane/oqtane.framework#1891] [Q&A]
After the site has pwa suport enabled, the javascript is displayed in web page instead of runing!
at the bottom of the webpage, the following is displaied:
<script>setTimeout(() => { var manifest = { "name": "db2Site", "short_name": "db2Site", "start_url": "http://core50.cloud123.net/db2/", "display": "standalone", "background_color": "#fff", "description": "db2Site", "icons": [{ "src": "http://core50.cloud123.net/db2/api/file/download/2", "sizes": "192x192", "type": "image/png" }, { "src": "http://core50.cloud123.net/db2/api/file/download/3", "sizes": "512x512", "type": "image/png" }] }; const serialized = JSON.stringify(manifest); const blob = new Blob([serialized], {type: 'application/javascript'}); const url = URL.createObjectURL(blob); document.getElementById('app-manifest').setAttribute('href', url); } , 1000);if ('serviceWorker' in navigator) { navigator.serviceWorker.register('/service-worker.js').then(function(registration) { console.log('ServiceWorker Registration Successful'); }).catch (function(err) { console.log('ServiceWorker Registration Failed ', err); }); };</script>

you can see live at :http://core50.cloud123.net/db2/mypage

**Comment by sbwalker** [comment]
fixed in #1896 - it was missing the @html.raw() encoding

**Reply by qunshen** [reply]
I modified _Host.cshtml and now the pwascript seems running. but I can see no pwa site features... the site just like normal site, no icon or splash screen....

**Reply by qunshen** [reply]
you can seee the wpa enabled site at:http://core50.cloud123.net/db2/mypage

**Reply by sbwalker** [reply]
#1898 should resolve the issue

**Reply by qunshen** [reply]
Thanks！  I am looking forward...
